src/CommunityToolkit.Maui/Behaviors/PlatformBehaviors/Touch/TouchBehavior.methods.shared.cs

```diff
@@ -63,7 +63,7 @@ internal async Task ForceUpdateState(CancellationToken token, bool animated = tr
 		}
 		catch (TaskCanceledException ex)
 		{
-			Trace.WriteLine(ex);
+			Trace.TraceError("{0}", ex);
```

Comment:
I think this should just be ex.Message to avoid spamming developers logs. A `TaskCanceledException` can be considered a valid scenario and therefore I don't think we need to log too much.

Reply:
@brminnick what do you think to these exceptions?

Reply:
Yup - 100% agree. We should avoid `Trace.Error` because tools like Sentry report it the same as an unhandled exception. 

We previously removed a `Trace.Error` in a previous PR from MediaElement for the same reason.

Reply:
Just to confirm this is **only** for `TaskCanceledException` right?